### DISCUSSION TITLE
Welcome to voicebox-pytorch Discussions!

### DISCUSSION THREAD

**Discussion by lucidrains** [lucidrains/voicebox-pytorch#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by manmay-nakhashi** [comment]
@lucidrains we can use aligner from naturalspeech2-pytorch over here it converges with the training after 50k steps. and then we can over sample each phonemes according to the alignment.

**Reply by lucidrains** [reply]
yes, was thinking about that! will look at it next week

signing off for park time with doggo 🐕  enjoy your Sunday!

**Reply by manmay-nakhashi** [reply]
just dumping it over here:
this can work with spectrogram and encodec.
https://github.com/charactr-platform/vocos

**Comment by lucidrains** [comment]
let's do this https://www.reddit.com/r/MachineLearning/comments/15lmvmc/d_im_losing_my_voice_due_to_illness_and_im/

**Reply by manmay-nakhashi** [reply]
i am in
